packages/playground/src/weblets/tf_contracts_list.vue

```diff
@@ -32,7 +32,22 @@
       </template>
 
       <template #[`item.state`]="{ item }">
-        <v-chip :color="getStateColor(item.value.state)">
+        <v-tooltip
+          v-if="item && item.value.state === ContractStates.GracePeriod"
+          :text="'Click here to view the amount you need to charge in order to unlock the contract.'"
```

Comment:
the amount of tokens needed to get your contract out of the grace period, to have your workloads functional again.